ai_guide/api_attractions/views.py

```diff
@@ -64,3 +66,17 @@ def create_attraction_obj(
                 attraction=attraction
             )
         return attraction
+
+
+class GetAudio(APIView):
+
+    def get(self, request, id):
+        attraction = get_object_or_404(Attraction, id=id)
+        polly = AwsPollyClient()
+        file_name = attraction.object_name
+        file_name = polly.get_audio(
+            file_name=file_name,
+            text=attraction.content
+        )
+        file = open(file_name, 'rb')
```

Comment:
what if something goes wrong with storing the content on hard drive?
We should probably handle potential exceptions here (or level above)